src/engine/errors.ts

```diff
@@ -14,6 +14,14 @@ export interface LimitsExceededContext {
   affectedEntities: AffectedEntities
 }
 
+export class AuthorizationErrorForApply extends ContentfulError {
```

Comment:
Do we need a dedicated authorization error for the apply command, or could both commands share the same error?

Reply:
We are not throwing this error in create-changeset yet, I would say as soon as we start to use it in both place we can adjust make it more generic, but atm the context that expects and the message is custom only for apply and cma token